### ISSUE TITLE
Screen Reader Support Specification Adaptation - Wizard Component

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#2666]
**Is your feature request related to a problem? Please describe.**
We should analyse the existing screen reader support implementation, along with the UI5 SR Specification itself and Aria Best Practices and adapt them in a way that fit the ui5 web components.

**Describe the solution you'd like**
Analyse and create a proposal specification for the wizard component. The screen reader support implementation proposal, should be documented in the accessibility status wiki of the component.

**Comment by elenastoyanovaa** [comment]
The implementation is analysed, the screen reader support should be re-done following the last recommendations and discussions. This will be done in https://github.com/SAP/ui5-webcomponents/issues/2729
